README.md

```diff
@@ -1,5 +1,8 @@
 # FirebaseUI for Android — UI Bindings for Firebase
 
+[![FirebaseOpensource.com](https://img.shields.io/badge/Documentation-FirebaseOpensource.com-orange.svg)](
```

Comment:
I like it!  But that's a big chunky badge.

Current: 
`Documentation: FirebaseOpensource.com`

Suggestion:
`Docs: firebaseopensource.com` (shorter, fewer caps)